### PR TITLE
Added link to Quobyte's volume plugin.

### PR DESCRIPTION
**Pull request by quolix**
Signed-off-by: Felix Hupfeld felix@quobyte.com

### CODE REVIEW

File: docs/extend/plugins.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ The following plugins exist:
 source volume plugin that provides multi-tenant, persistent, distributed storage
 with intent based consumption using ceph underneath.
 
+* The [Quobyte Volume Plugin](https://github.com/quobyte/docker-volume) connects Docker to [Quobyte](http://www.quobyte.com/containers)'s data center file system, a general-purpose scalable and fault-tolerant storage platform.
```
Comment (thaJeztah) [inline docs/extend/plugins.md]: Can you move this above the "REX-Ray" plugin? I realize the list is already out of order, but we should try to keep it sorted alphabetically

After moving, please commit with `git commit --amend`, and do a *force* push (`git push -f ...`), that should update the PR on github

Comment (quofelix) [inline docs/extend/plugins.md]: Done.

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by vdemeester** [comment]
LGTM 🐵

**Pull request merged**